packages/block-library/src/latest-posts/edit.js

```diff
@@ -77,6 +80,8 @@ function getFeaturedImageDetails( post, size ) {
 
 export default function LatestPostsEdit( { attributes, setAttributes } ) {
 	const instanceId = useInstanceId( LatestPostsEdit );
+	const dropdownMenuProps = useToolsPanelDropdownMenuProps(); // Moved inside the component
```

Comment:
```suggestion
	const dropdownMenuProps = useToolsPanelDropdownMenuProps();
```

I think this comment is redundant.

Reply:
@t-hamano - All changes done. Thank you for the review.